### PR TITLE
Update motor tests to work with BrickPi

### PR DESCRIPTION
**Pull request by ddemidov**
This resolves merge conflicts in #152.

### CODE REVIEW

**Comment by ddemidov** [comment]
@dlech, could you please check if this still does what was intended originally?

**Comment by dlech** [comment]
Yes. Ping me if I haven't done this in a week or two.

**Comment by ddemidov** [comment]
> Ping me if I haven't done this in a week or two.

ping

**Comment by dwalton76** [comment]
@ddemidov two more sync conflicts have cropped up...can you take a quick look and we can get this in?

**Comment by ddemidov** [comment]
I've rebased on current develop, but I can not test the changes on brickpi (don't have it).

**Comment by dwalton76** [comment]
me neither mine died a while back :( @dlech can you give this a quick test and let us know if it looks good?

File: tests/motor/motor_param_unittest.py [hunk 1]
```diff
@@ -490,49 +567,73 @@ def test_time_sp_large_positive(self):
...
+    'motor': ev3.Motor('pistorms:BAM1'),
+    'port': 'pistorms:BAM1',
+    'driver_name': 'lego-nxt-motor',
+    'commands': ['run-forever', 'run-to-abs-pos', 'run-to-rel-pos', 'run-timed', 'stop', 'reset'],
+    'stop_actions': ['coast', 'brake', 'hold'],
+    'supports_encoder_polarity': False,
+    'speed_pid': { 'kP': 1000, 'kI': 60, 'kD': 0 },
+    'hold_pid': { 'kP': 20000, 'kI': 0, 'kD': 0 },
+}
+paramsA = pistorms_params
```
Comment (dlech) [inline tests/motor/motor_param_unittest.py]: This should probably be `paramsA = ev3_params`.

**Comment by dlech** [comment]
I updated my brickpi branch with some more commits, but #152 seems to have ignored it since it is closed and it won't let me re-open.

https://github.com/dlech/ev3dev-lang-python/tree/brickpi

**Comment by ddemidov** [comment]
I've pulled commits from https://github.com/dlech/ev3dev-lang-python/tree/brickpi here.

File: tests/motor/motor_param_unittest.py (hunk 1 above)
Comment (ddemidov) [reply tests/motor/motor_param_unittest.py]: Is this still an issue?

**Comment by WasabiFan** [comment]
What's the situation here: do we want to resolve conflicts and merge? I'm happy with this if you guys believe it doesn't break anything.

**Comment by dwalton76** [comment]
+1 for merging it

**Comment by ddemidov** [comment]
I've just reset contents of tests/motor to that of 'brickpi' branch at dlech/ev3dev-lang-python. This narrows the PR to this particular folder and (I hope) makes sure we don't screw up other parts of the repo.

**Comment by dwalton76** [comment]
I don't have a brickpi but I'll at least test out the patch tonight to make sure tests are ok on EV3 and EVB...assuming so will merge it after that.

**Pull request merged**